services/app-api/forms/naaar.json

```diff
@@ -1081,19 +1081,49 @@
               "type": "p",
               "children": [
                 {
-                  "type": "p",
-                  "content": "This program is missing required information. You won’t be able to access this section until you’ve completed all the required questions in section "
-                },
-                {
-                  "type": "internalLink",
-                  "content": "I. State and program information",
-                  "props": {
-                    "to": "/naaar/state-and-program-information/state-information-and-reporting-scenario"
-                  }
+                  "type": "span",
+                  "content": "This program is missing required information. You won’t be able to access this section until you’ve completed all the required questions in section I."
                 },
                 {
-                  "type": "span",
-                  "content": "."
+                  "type": "ul",
```

Comment:
Now the `ul` is [not allowed to be a child](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/p#:~:text=Permitted%20content,Phrasing%20content) of `p`. Can they be siblings?

The top node could be a `div`, with the `span` turning into your `p` and that would make the `p` and `ul` siblings.

Incorrect HTML output:
![Managed Care Reporting 2025-05-08 at 14 56 45@2x](https://github.com/user-attachments/assets/8e348131-c710-4ea9-ac99-8dda93d01864)

Reply:
Ah, I see, I misunderstood @bangbay-bluetiger's suggestion, which was exactly that. Just fixed!